packages/gestalt/src/Video.js

```diff
@@ -495,57 +496,59 @@ export default class Video extends React.PureComponent<Props, State> {
         className={styles.player}
         style={{ paddingBottom, height: fullscreen ? '100%' : 0 }}
       >
-        <video
-          autoPlay={playing}
-          loop={loop}
-          muted={volume === 0}
-          playsInline={playsInline}
-          poster={poster}
-          preload={preload}
-          src={typeof src === 'string' ? src : undefined}
-          ref={this.setVideoRef}
-          className={styles.video}
-          onCanPlay={this.handleCanPlay}
-          onDurationChange={this.handleDurationChange}
-          onEnded={this.handleEnded}
-          onSeeked={this.handleSeek}
-          onTimeUpdate={this.handleTimeUpdate}
-          onProgress={this.handleProgress}
-        >
-          {Array.isArray(src) &&
-            src.map(source => (
-              <source key={source.src} src={source.src} type={source.type} />
-            ))}
-          <track kind="captions" src={captions} />
-        </video>
-        {children && (
-          <Box position="absolute" top left bottom right overflow="hidden">
-            {children}
-          </Box>
-        )}
-        {/* Need to use full path for these props so Flow can infer correct subtype */}
-        {this.props.controls && (
-          <VideoControls
-            accessibilityMaximizeLabel={this.props.accessibilityMaximizeLabel}
-            accessibilityMinimizeLabel={this.props.accessibilityMinimizeLabel}
-            accessibilityMuteLabel={this.props.accessibilityMuteLabel}
-            accessibilityPauseLabel={this.props.accessibilityPauseLabel}
-            accessibilityPlayLabel={this.props.accessibilityPlayLabel}
-            accessibilityUnmuteLabel={this.props.accessibilityUnmuteLabel}
-            currentTime={currentTime}
-            duration={duration}
-            fullscreen={fullscreen}
-            onPlay={this.handlePlay}
-            onPlayheadDown={this.handlePlayheadDown}
-            onPlayheadUp={this.handlePlayheadUp}
-            onPause={this.handlePause}
-            onFullscreenChange={this.toggleFullscreen}
-            onVolumeChange={this.handleVolumeChange}
-            playing={playing}
-            seek={this.seek}
-            volume={volume}
-          />
-        )}
+        <GestaltProvider id="Video" colorScheme="light">
```

Comment:
Only change is line 499

Reply:
Instead of using `GestaltProvider` here, could you just specifically use `ColorSchemeProvider` instead? We'll eventually have other contexts set through `GestaltProvider` unrelated to colors (maybe RTL or routing?) so for internal use like this it's more clear to just update the color scheme